File: source/frontend/src/app/entities/vacature/werkgever.ts
import {AdresHouding} from './adresHouding';
import {ContactpersoonAfdeling} from './contactpersoonAfdeling';
import {SectorBeroepsEnBedrijfsleven} from '../shared/sectorBeroepsEnBedrijfsleven';
import {Webadres} from '../shared/webadres';

export interface Werkgever {
  handelsnaamOrganisatie?: string;
  webadres?: Array<Webadres>;
  adresHouding?: Array<AdresHouding>;
  contactpersoon?: Array<ContactpersoonAfdeling>;
  sector?: Array<SectorBeroepsEnBedrijfsleven>;
}

File: source/backend-vacatures-bemiddelaar/src/main/java/com/cap/pocvng/entity/Werkgever.java
package com.cap.pocvng.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.validation.Valid;
import javax.validation.constraints.Size;
import java.util.List;

/**
 * Werkgever
 */


@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Werkgever {

    @Size(max = 500)
    private String handelsnaamOrganisatie;

    @Valid
    private List<Webadres> webadres;

    @Valid
    private List<AdresHouding> adresHouding;

    @Valid
    private List<ContactpersoonAfdeling> contactpersoon;

    @Valid
    private List<SectorBeroepsEnBedrijfsleven> sector;

}
